tests/ui/suggestions/issue-114701.rs

```diff
@@ -0,0 +1,15 @@
+enum Enum<T> { , SVariant { v: T }, UVariant } //~ ERROR expected identifier, found `,`
```

Comment:
Are these extra errors required for reproducing the issue?
If not, then all "off-topic" errors are better removed from the test.